src/entities/Planet.ts

```diff
@@ -18,3 +18,5 @@ export const schema = new EntitySchema<Planet, BaseEntity>({
     population: { type: 'number' },
   },
 });
+
+exports.planetSchema = schema;
```

Comment:
The goal was not have to add the export per file and use the index.ts as an auto export like Java does by default.

Reply:
Yeah, so I wasn't sure if there was some magic going on with `mikro-orm` that needed the variable to be called `schema`, so this only changes the name under which that reference will be exported to other modules.

Without the name change, the `typescript` compiler was complaining that your `src/entities/index.ts` was exporting two variables which were both called `schema` (name conflict). This was because both `Planet.ts` and `BaseEntity.ts` were exporting variables called `schema`.

So, there are probably other ways around that issue than the one I chose here.